### PR TITLE
document SC 1.1 compatibility with different s2i wrappers

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
Closes https://github.com/SeldonIO/seldon-core/issues/1601

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ryandawsonuk**

If they are not already assigned, you can assign the PR to them by writing `/assign @ryandawsonuk` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Wed Mar 25 15:39:58 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1602/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1602 --build=2

**Comment by seldondev** [comment]
Wed Mar 25 15:40:05 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1602/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1602 --build=1

**Comment by seldondev** [comment]
Wed Mar 25 15:58:11 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1602/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1602 --build=3

**Comment by seldondev** [comment]
Wed Mar 25 15:59:28 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1602/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1602 --build=4

**Comment by seldondev** [comment]
Wed Mar 25 16:15:44 UTC 2020
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1602/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1602 --build=5

**Comment by seldondev** [comment]
Wed Mar 25 16:17:11 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1602/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1602 --build=6

**Comment by seldondev** [comment]
Wed Mar 25 16:35:28 UTC 2020
The logs for [lint] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1602/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1602 --build=8

**Comment by seldondev** [comment]
Wed Mar 25 16:35:29 UTC 2020
The logs for [pr-build] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1602/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1602 --build=7

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1602#issuecomment-603964404" title="Approved">axsaucedo</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: doc/source/reference/upgrading.md [hunk 1]
```diff
@@ -27,6 +27,43 @@ For those wishing to use the deprecated Java engine service orchestrator see [th
...
+Model has been deployed both with REST and GRPC API with both new orchestrator and the deprecated Java engine.
+Test verifies if model can successfully serve inference requests.
+
+**NOTE:** Full support of custom metrics and tags with new orchestrator is only available from Python wrapper version 0.19.
+If you need to use older version of Python wrapper you can continue to use Java engine as described above until the next release.
+
+
+| Language Wrapper |     Version   | API Type | New Orchestrator  | Deprecated Java engine | Notes                                   |
+|------------------|---------------|----------|-------------------|------------------------|-----------------------------------------|
+| Python           | 0.19          | both     | yes               | yes                    | full support of custom metrics and tags |
```
Comment (adriangonz) [inline doc/source/reference/upgrading.md]: nitpick: would it make it clearer if we used ✅ and ❌ instead of `yes` and `no`?

**Comment by seldondev** [comment]
Wed Mar 25 18:16:34 UTC 2020
The logs for [lint] [10] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1602/10.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1602 --build=10

**Pull request merged**